x-pack/plugins/canvas/server/usage/__tests__/workpad_collector.handle_response.ts

```diff
@@ -117,7 +118,7 @@ describe('usage collector handle es response data', () => {
   });
 
   it('should fail gracefully in general', () => {
-    const usage = handleResponse({ hits: { total: 0 } });
-    expect(usage).to.eql(undefined);
+    const usage = handleResponse({ hits: { total: 0 } } as any);
```

Comment:
This line would then change, as well... it should match `SearchResponse<WorkpadSearch>`, right?